misk-policy/build.gradle.kts

```diff
@@ -17,6 +17,8 @@ dependencies {
   api(Dependencies.retrofit)
   api(project(":misk-core"))
   api(project(":misk-inject"))
+  api(project(":misk-metrics"))
+  implementation(Dependencies.prometheusClient)
```

Comment:
does this not come down through :misk-mtrics already?

Reply:
just curious

Reply:
The linter forced me to import it explicitly. Seems odd.